src/Organisms/NotFoundContent/NotFoundPageWrapperStyled.js

```diff
@@ -0,0 +1,31 @@
+import styled from 'styled-components';
+import FONTS from '../../Styles/fontsStyledComponents';
+const NotFoundPageWrapperStyled = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 51px 16px;
+
+  .mainContentWrapper {
+    display: flex;
+    width: 16.25rem;
+    height: 7.5rem;
+    flex-direction: column;
+    justify-content: space-between;
+    margin: 35px 0 68px 0;
```

Comment:
There is no need for last 0

Reply:
The same as yesterday ;) For me it's more readable, I don't remember all shortland form :)

Reply:
from developer.mozilla.org:
![image](https://user-images.githubusercontent.com/92270179/151589249-1af92c2c-56c3-4c10-9998-4a685d057b7f.png)
![image](https://user-images.githubusercontent.com/92270179/151589302-30856c9d-5c6d-4bf5-a7b3-e42a4b877310.png)
if the Idea here is to have same margins at left and right side, like examples - 3 values are enough, and has exactly that meaning

Reply:
But it's not telling, that I can't use 4 values... But ok.